lib/python/treadmill_aws/bootstrap/zookeeper/aws.linux/treadmill/init/acl_sync.yml

```diff
@@ -0,0 +1,12 @@
+user: "{{ treadmillid }}"
+command: |
+  echo \
+    {{ treadmill }}/bin/treadmill \
+        sproc srvacl /treadmill/roles/servers
```

Comment:
Since you are running a `{{treadmillid}}` (this is the right thing), you'll need to chown the roles directory too